package.json

```diff
@@ -16,8 +16,9 @@
   "scripts": {
     "clean": "rimraf lib",
     "watch": "yarn rollup --watch",
-    "build": "npm run clean && babel src --out-dir lib",
+    "build": "npm run clean && babel src --out-dir lib --extensions '.ts,.tsx,.js,.jsx'",
```

Comment:
Curious why we include the `jsx` extension here but not in the extensions list in the other config files.

Reply:
missed it in the other file, added